system.c

```diff
@@ -176,6 +176,14 @@ uint8_t system_execute_line(char *line)
           else { report_ngc_parameters(); }
           break;
 
+        case 'P': // Set force sensitivity parameter. Avoids hastle of changing eeprom settings.
+          char_counter++;
+          if (!read_float(line, &char_counter, &parameter)){
```

Comment:
Oh! I think I get it now. The data coming in is an ASCII representation of the number, so that a value like 130, instead of being 0x82, is really 0x313330 ("130"). 

LGTM.